### PR TITLE
Translates to portuguese

### PR DESCRIPTION
**Pull request by 9thScientist**
This PR should be able to translate parts of the website to Portuguese. Namely, the `user` attributes and error messages.

### CODE REVIEW

File: config/locales/pt.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pt:
+  activerecord:
+    attributes:
+      user:
+        name: "Nome"
+        submit: "Cenas"
```
Comment (frm) [inline config/locales/pt.yml]: Huh?

**Review by frm — CHANGES_REQUESTED** [review]

File: config/locales/pt.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    errors:
+      messages:
+        blank: "Não pode estar em branco"
+        invalid: "É inválido"
```
Comment (frm) [inline config/locales/pt.yml]: We're missing "has already been taken". It should be something como "Já está inscrito"

**Review by frm — CHANGES_REQUESTED** [review]

File: config/locales/pt.yml (hunk 1 above)
Comment (9thScientist) [reply config/locales/pt.yml]: Sorry. I was testing things out and didn't noticed that I left this here.

File: config/locales/pt.yml (hunk 2 above)
Comment (9thScientist) [reply config/locales/pt.yml]: Yeah you are right. Maybe the translation should be "Utilizador já existe"

**Review by frm — APPROVED** [review]

**Pull request merged**